static/css/material.less

```diff
@@ -14,19 +14,31 @@
   }
   .card-title {
     margin-bottom: 5px;
-    margin-top: 20px;
+    margin-top: 10px;
     padding-right: 15px;
     padding-left: 15px;
+    div, h3 {
+      display: inline-block;
+    }
+    i {
+      font-size: 2.143em;
```

Comment:
I think we can probably round to the nearest 1 decimal place. Try "2.1" or "2.15" if absolutely necessary. This will make absolutely no difference in the UI but will be much easier to read.